packages/admin/admin/package.json

```diff
@@ -12,7 +12,8 @@
     "main": "lib/index.js",
     "types": "lib/index.d.ts",
     "files": [
-        "lib"
+        "lib",
+        "assets"
```

Comment:
Technically, this should work. But that's as far as i can see, the first image that we want ship with the npm package. 

Possible disadvantages:
- raise package size of `@comet/admin`

Reply:
Our main admin bundle.js has in a gzip version -> 2 MB - far away from 300 KB. 😅, but it's not a public site. Anyways i think we should not make the bundle bigger than needed (if we can avoid it)

I definitely agree that an SVG would be better in all terms. I already requested an SVG version, hope that we will get one soon.